frontend/src/routes/question/QuestionPage.vue

```diff
@@ -0,0 +1,108 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import { useRouter, useRoute } from 'vue-router';
+import QuestionBottomBar, { type IButton } from './QuestionBottomBar.vue';
+import QuestionCard from './QuestionCard.vue';
+import NavBar from '../../components/NavBar.vue';
+import { appRoutes } from '@/main';
+
+const router = useRouter();
+const route = useRoute();
+const questionNr = computed(() => parseInt(route.params['nr'] as string));
+const questionProgress = ref(questionNr.value);
+const questionTotal = 10;
+const handleClick = (type: 'yes' | 'no' | 'skip') => {
```

Comment:
Tohle (asi?) se chová divně, když už mám vše vyplněno, vrátím se někam doprostřed, kliknu na něco a hodí mě to rovnou na rekapitulaci. To mi přijde divný, nevím…

Reply:
to se musi pak doresit az se storem. stejne by se to pak menilo

Reply:
uz chapu. opraveno v dalsim commitu